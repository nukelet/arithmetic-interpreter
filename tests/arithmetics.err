Error parsing expression: 2 [@] 1
Error parsing expression: 1 [1]
Error parsing expression: [+] 3
Traceback (most recent call last):
  File "/mnt/storage/DATA/projects/interpreter/arithmetic_interpreter.py", line 217, in <module>
    main()
  File "/mnt/storage/DATA/projects/interpreter/arithmetic_interpreter.py", line 201, in main
    text = input()
EOFError: EOF when reading a line
