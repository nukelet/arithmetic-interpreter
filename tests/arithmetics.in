2 @ 1
1 1
+ 3
1 + 2
20 + 1
1 - 2
22 - 2
3 *4
4/   3
2 + 3 * 2 + 3 / 2 - 1