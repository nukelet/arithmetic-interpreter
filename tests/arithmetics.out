


3
21
-1
20
12
1
8
